structs.go

```diff
@@ -623,6 +641,10 @@ func requestStructForOp(op int32) interface{} {
 		return &SetDataRequest{}
 	case opSetWatches:
 		return &setWatchesRequest{}
+	case opSetWatches2:
```

Comment:
A nicer name would be good here.

Reply:
this is actually the name of the op in zk. blame java i guess?